compiler/src/dmd/dtemplate.d

```diff
@@ -4556,9 +4556,15 @@ extern (C++) class TemplateInstance : ScopeDsymbol
 
         // The arguments are not treated as part of a default argument,
         // because they are evaluated at compile time.
+        const inCondition = !!(sc.flags && SCOPE.condition);
+        // For master branch: const inCondition = sc.condition;
```

Comment:
Kind of annoying that this refactoring isn't in stable yet, does a comment like this help when merging stable @ibuclaw ?